include.mk

```diff
@@ -2,7 +2,7 @@ all:
 
 # Compiler options here.
 ifeq ($(USE_OPT),)
-  USE_OPT = -Os -ggdb --specs=nosys.specs -lnosys -lm -ffast-math
+  USE_OPT = -O1 -ggdb -std=gnu99 --specs=nano.specs -lm -ffast-math -DCHPRINTF_USE_FLOAT=1
```

Comment:
For me, -lnosys is necessary - otherwise I get the following error:
```
/home/jschall/Downloads/gcc-arm-none-eabi-6-2017-q2-update/bin/../lib/gcc/arm-none-eabi/6.3.1/../../../../arm-none-eabi/lib/thumb/v7e-m/fpv4-sp/hard/libg_nano.a(lib_a-sbrkr.o): In function `_sbrk_r':
sbrkr.c:(.text._sbrk_r+0xc): undefined reference to `_sbrk'
```